core/unit_test/TestWithoutInitializing.hpp

```diff
@@ -104,7 +104,7 @@ TEST(TEST_CATEGORY, realloc_exec_space) {
       [&](BeginFenceEvent event) {
         if ((event.descriptor().find("Debug Only Check for Execution Error") !=
              std::string::npos) ||
-            (event.descriptor().find("HostSpace fence") != std::string::npos))
+            (event.descriptor().find(std::string(TEST_EXECSPACE{}.name()).append(": ZeroMemset for")) != std::string::npos))
```

Comment:
I think the string descriptor and the type being matched against don't line up now?

Reply:
I'm not sure to understand what you mean...

Reply:
I'll close this thread because I reverted the changes :no_entry: